### ISSUE TITLE
Use whatthefee.io as a mining fee rate prediction service

### ISSUE THREAD

**Issue by cbeams** [bisq-network/bisq-pricenode-deprecated#8]
We currently use https://bitcoinfees.earn.com as a mining fee rate prediction service. A new service is now available at https://whatthefee.io.

BitcoinFees has not been ideal, as its "recommended" fees are usually far too high. We've mitigated this with our own custom logic, but a more reliable / accurate service is still desirable.

This issue, then, is about looking into the feasibility of using this new service, including possibly running our own instance(s) of it, and swapping out our use of BitcoinFees for it.

For details, see https://github.com/cbeams/bisq-pricenode/blob/refactor/src/main/java/bisq/price/mining/providers/BitcoinFeeRateProvider.java (still in a PR branch, but this is what will soon be on master).

**Comment by ManfredKarrer** [comment]
Maybe you can reach out to https://twitter.com/FelixWeis and ask him if he would be interested to contribute?

**Comment by cbeams** [comment]
Yeah, I'll do that after we've merged the PR in #7, so it's that much more straightforward to see / explain what needs to be done.

**Comment by cbeams** [comment]
@0xb10c, if you're interested in looking into this, the `master` branch of this repository now represents what's in production (with today's merge of #7).

I'd say the first step would be to put together a pull request that implements an alternative to the existing `BitcoinFeeRateProvider` class based on the whatthefee.io API instead of earn.com. We can discuss and go from there in order to decide whether we should fully replace the provider implementation, or stack them up in a primary / secondary fallback arrangement.

**Comment by 0xB10C** [comment]
Just started setting up a local pricenode. Will take a closer look at the code in the comming days. Thanks!

**Comment by 0xB10C** [comment]
I'm on a tight schedule right now. Probably won't be starting soon. This issue is open for takers.

**Comment by cbeams** [comment]
Thanks for letting everyone know @0xB10C.
